### ISSUE TITLE
Avoid positional CLI parameters

### ISSUE THREAD

**Issue by ptrebert** [core-unit-bioinformatics/devtools#12]
That way of defining CLI parameters makes them positional. If used at all, positional parameters should be limited to a low number like two, and only if a "first that than that" is somewhat established in the field (cf.: genomic aligners often have two positional parameters for (first) reference and (second) reads). Making all parameters positional makes it hard to use the script (have to remember the order), and command lines read "on-the-fly" are hard to parse because appropriately named options (`--reads`, `--project-dir`) provide contextual meaning.

https://github.com/core-unit-bioinformatics/devtools/blob/ab539f8b135350189a99627fbf45229df0297c01/update_template.py#L30

Another point: in the Linux world, longer names for CLI parameters are separated by `-` (dash/minus), see for instance the help of `git` (e.g., `--no-hardlinks` or `--[no-]reject-shallow`) or `grep` (e.g., `-v, --invert-match`). There is a simple reason for that: instead of two keystrokes (shift + minus), it only requires one. That is a hard argument.

**Comment by svenbioinf** [comment]
updated

**Comment by svenbioinf** [comment]
removed positional parameters in 2e4e61e24953941425373bba52baed267dbd9723
